test/testinput/bufr_ncep_adpsfc.yaml

```diff
@@ -0,0 +1,277 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+# #
+# # This software is licensed under the terms of the Apache Licence Version 2.0
+# # which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "testinput/gdas.t12z.adpsfc_thinned.tm00.bufr_d"
+
+      exports:
+        variables:
+          timestamp:
+            datetime:
+              year: "*/YEAR[1]"
+              month: "*/MNTH[1]"
+              day: "*/DAYS[1]"
+              hour: "*/HOUR[1]"
+              minute: "*/MINU[1]"
+          altimeterSettingQnh:
+            query: "*/ALSE"
+          cloudAmount:
+            query: "[*/CLDSQ1/CLAM, */MTRCLD/CLAM, NC000100/CLAM[1], NC000101/CLAM[1], NC000102/CLAM[1]]"
```

Comment:
I don't think we need to index for CLAM. 
This is my suggestion:
          cloudAmount:
            query: "[*/CLAM, */CLDSQ1/CLAM, */MTRCLD/CLAM]"

Reply:
I mostly do it to be safe but I'll put it as this.